apps/protocol-frontend/src/components/ContributionsTableShell.tsx

```diff
@@ -22,28 +22,32 @@ import PageHeading from './PageHeading';
 import ContributionsTable from './ContributionsTable';
 import ContributionTypesTable from './ContributionTypesTable';
 import EmptyContributions from './EmptyContributions';
-import { useContributions } from '../contexts/ContributionContext';
+import { useUser } from '../contexts/UserContext';
+import { useContributionInfiniteList } from '../hooks/useContributionList';
+import { UIContribution } from '@govrn/ui-types';
+import { Row } from 'react-table';
+import { ContributionTableType } from '../types/table';
 
 const ContributionsTableShell = () => {
-  const { isUserContributionsLoading, userContributions } = useContributions();
+  const { userData } = useUser();
+  const {
+    data: contributions,
+    isFetching,
+    hasNextPage,
+    fetchNextPage,
+  } = useContributionInfiniteList({
+    where: {
+      user_id: { equals: userData?.id },
```

Comment:
Nice. I love that we're able to pass all the args into the hook itself as needed